ghost/core/test/unit/frontend/public/ghost-stats.test.js

```
@@ -2,15 +2,96 @@ const fs = require('fs');
+    const source = urlObj.searchParams.get('source');
+    if (source) {
+        return source;
+    }
+    
+    const utmSource = urlObj.searchParams.get('utm_source');
+    if (utmSource) {
+        return utmSource;
+    }
+    
+    // Special case for portal hash URLs
+    if (urlObj.hash && urlObj.hash.includes('/portal')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)